src/styles/main.scss

```diff
@@ -0,0 +1,87 @@
+$calendar-day-size: 100px;
+$calendar-border: 1px;
+$calendar-color-default: #eee;
+$calendar-hover-color: #ffbfcb;
+$calendar-padding: 10px;
+$calendar-gap: 1px;
+$font-size: 30px;
+$animation-duration: 0.5s;
+$first-day-margin: 606px;
+$calendar-width: 726px;
+$week-days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+body,
+html,
+div {
+  height: 100%;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  place-content: center start;
+  padding: $calendar-padding;
+  gap: $calendar-gap;
+  width: 726px;
```

Comment:
The width of the `.calendar` class is hardcoded as `726px`. It would be better to use the `$calendar-width` variable here for consistency and maintainability. Consider updating this line to `width: $calendar-width;`.